src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -214,907 +214,908 @@ void GetPurchasedAir(EnergyPlusData &state)
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name,
+                            PurchAir.Name,
                             Constant::eResource::DistrictHeatingWater,
                             OutputProcessor::Group::HVAC,
                             OutputProcessor::EndUseCat::Heating);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Sensible Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).SenCoolEnergy,
+                            PurchAir.SenCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Latent Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).LatCoolEnergy,
+                            PurchAir.LatCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Total Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).TotCoolEnergy,
+                            PurchAir.TotCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name,
+                            PurchAir.Name,
                             Constant::eResource::DistrictCooling,
                             OutputProcessor::Group::HVAC,
                             OutputProcessor::EndUseCat::Cooling);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Supply Air Total Heating Fuel Energy",
+                            Constant::Units::J,
+                            PurchAir.TotHeatFuelEnergy,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Sum,
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Supply Air Total Cooling Fuel Energy",
+                            Constant::Units::J,
+                            PurchAir.TotCoolFuelEnergy,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Sum,
+                            PurchAir.Name);
```

Comment:
Added two energy consumption report variables for supply air: `Zone Ideal Loads Supply Air Total Heating Fuel Energy` and `Zone Ideal Loads Supply Air Total Cooling Fuel Energy`.